pkg/registry/serviceaccount/etcd/etcd_test.go

```diff
@@ -70,7 +70,7 @@ func TestUpdate(t *testing.T) {
 		// updateFunc
 		func(obj runtime.Object) runtime.Object {
 			object := obj.(*api.ServiceAccount)
-			// TODO: Update this serviceAccount
+			object.Secrets = []api.ObjectReference{api.ObjectReference{}}
```

Comment:
ah. might want to make the objectreference contain a name... surprised it passed validation without it